sparkle/src/components/IconButton.tsx

```diff
@@ -1,16 +1,17 @@
+import { SIDE_OPTIONS } from "@radix-ui/react-popper";
 import React, { ComponentType, MouseEventHandler } from "react";
 
 import { classNames } from "@sparkle/lib/utils";
 
 import { Icon, IconProps } from "./Icon";
-import { Tooltip, TooltipProps } from "./Tooltip";
+import { Tooltip } from "./Tooltip";
 
 type IconButtonProps = {
   variant?: "primary" | "warning" | "secondary" | "tertiary" | "white";
   onClick?: MouseEventHandler<HTMLButtonElement>;
   size?: "xs" | "sm" | "md";
   tooltip?: string;
-  tooltipPosition?: TooltipProps["position"];
+  tooltipPosition?: (typeof SIDE_OPTIONS)[number];
```

Comment:
`nit` but I'd avoid spreading the underlying implementation of the `Tooltip` and would use `React.ComponentProps<Tooltip>["side"]`